tianshou/policy/modelfree/pg.py

```diff
@@ -20,69 +23,75 @@
 class PGPolicy(BasePolicy):
     """Implementation of REINFORCE algorithm.
 
-    :param torch.nn.Module model: a model following the rules in
-        :class:`~tianshou.policy.BasePolicy`. (s -> logits)
-    :param torch.optim.Optimizer optim: a torch.optim for optimizing the model.
+    :param actor: mapping (s->model_output), should follow the rules in
+        :class:`~tianshou.policy.BasePolicy`.
+    :param optim: optimizer for actor network.
     :param dist_fn: distribution class for computing the action.
-    :param float discount_factor: in [0, 1]. Default to 0.99.
-    :param bool action_scaling: whether to map actions from range [-1, 1] to range
-        [action_spaces.low, action_spaces.high]. Default to True.
-    :param str action_bound_method: method to bound action to range [-1, 1], can be
-        either "clip" (for simply clipping the action), "tanh" (for applying tanh
-        squashing) for now, or empty string for no bounding. Default to "clip".
-    :param Optional[gym.Space] action_space: env's action space, mandatory if you want
-        to use option "action_scaling" or "action_bound_method". Default to None.
-    :param lr_scheduler: a learning rate scheduler that adjusts the learning rate in
-        optimizer in each policy.update(). Default to None (no lr_scheduler).
-    :param bool deterministic_eval: whether to use deterministic action instead of
-        stochastic action sampled by the policy. Default to False.
+        Maps model_output -> distribution. Typically a Gaussian distribution
+        taking `model_output=mean,std` as input for continuous action spaces,
+        or a categorical distribution taking `model_output=logits`
+        for discrete action spaces. Note that as user, you are responsible
+        for ensuring that the distribution is compatible with the action space.
+    :param action_space: env's action space.
+    :param discount_factor: in [0, 1].
+    :param reward_normalization: if True, will normalize the *returns*
+        by subtracting the running mean and dividing by the running standard deviation.
+        Can be detrimental to performance! See TODO in process_fn.
+    :param deterministic_eval: if True, will use deterministic action (the dist's mode)
+        instead of stochastic one during evaluation. Does not affect training.
+    :param observation_space: Env's observation space.
+    :param action_scaling: if True, scale the action from [-1, 1] to the range
+        of action_space. Only used if the action_space is continuous.
+    :param action_bound_method: method to bound action to range [-1, 1].
+        Only used if the action_space is continuous.
+    :param lr_scheduler: if not None, will be called in `policy.update()`.
 
     .. seealso::
 
-        Please refer to :class:`~tianshou.policy.BasePolicy` for more detailed
-        explanation.
+        Please refer to :class:`~tianshou.policy.BasePolicy` for more detailed explanation.
```

Comment:
could you double-check if the rendered html doc has any formatting issues locally?

Reply:
I went over it, looks good to me, links are resolved